packages/esm-patient-chart-app/src/ui-components/action-menu.component.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useCallback } from 'react';
+import { ExtensionSlot, useLayoutType } from '@openmrs/esm-framework';
+import Button from 'carbon-components-react/es/components/Button';
+import { HeaderPanel } from 'carbon-components-react';
+
+import { Edit20 } from '@carbon/icons-react';
+import styles from './action-menu.scss';
+import { isDesktop } from '../utils';
+
+interface ActionMenuInterface {
+  open: boolean;
+}
+
+export const CHARTS_DRAWER_SLOT = 'drawer-slot';
+export const CHARTS_ACTION_MENU_ITEMS_SLOT = 'action-menu-items-slot';
+
+export const ActionMenu: React.FC<ActionMenuInterface> = ({ open }) => {
+  const layout = useLayoutType();
+
+  const menu = isDesktop(layout) ? (
+    <aside className={styles.rightSideNav}>
+      <ExtensionSlot extensionSlotName={CHARTS_ACTION_MENU_ITEMS_SLOT} />
+    </aside>
+  ) : (
+    <button className={styles.actionBtn}>
```

Comment:
Should this be a single button or also an extension slot?

Reply:
dont know honestly - seems like a singular button will open this mobile drawer thing. it can be hardcoded i suppose. i just wonder if the drawer on tablet is supposed to be entirely different from the one on desktop

Reply:
Yes, sorry from the symbol (edit?) it wasn't looking like just for toggling the drawing. All good!